types/types.go

```diff
@@ -134,22 +133,22 @@ type RuleErrorKey struct {
 
 // RuleWithContent represents a rule with content, basically the mix of rule and rule_error_key tables' content
 type RuleWithContent struct {
-	Module       RuleID          `json:"module"`
-	Name         string          `json:"name"`
-	Summary      string          `json:"summary"`
-	Reason       string          `json:"reason"`
-	Resolution   string          `json:"resolution"`
-	MoreInfo     json.RawMessage `json:"more_info"`
-	ErrorKey     ErrorKey        `json:"error_key"`
-	Condition    string          `json:"condition"`
-	Description  string          `json:"description"`
-	TotalRisk    int             `json:"total_risk"`
-	RiskOfChange int             `json:"risk_of_change"`
-	PublishDate  time.Time       `json:"publish_date"`
-	Active       bool            `json:"active"`
-	Internal     bool            `json:"internal"`
-	Generic      string          `json:"generic"`
-	Tags         []string        `json:"tags"`
+	Module       RuleID    `json:"module"`
+	Name         string    `json:"name"`
+	Summary      string    `json:"summary"`
+	Reason       string    `json:"reason"`
+	Resolution   string    `json:"resolution"`
+	MoreInfo     string    `json:"more_info"`
```

Comment:
We already had `MoreInfo` as string and it brings a lot of problems. So are you sure it should be string?

Reply:
What problems? I was not aware of them.
Using json.RawMessage for a field that won't contain a json is weird, and in fact, is causing a problem when trying to access the content (take a look to the bug intended to be solved by this PR)

Reply:
Other option could be to send it as `byte[]` instead of string, but the content of the rest of markdown files (like description, reason, summary, resolution...) is already handled as string

Reply:
The problem was that JSON wasn't parsed and markdown didn't work, we can add a fix somewhere else though, like in smart proxy maybe